spec/system/signup/signup_spec.rb

```diff
@@ -0,0 +1,49 @@
+require_relative "../../../lib/google_mail"
+require_relative "../../../lib/eapol_test"
+
+feature "Signup" do
+  include_context "signup"
+
+  it "signs up successfully" do
+    gmail = GoogleMail.new
+
+    test_email = gmail.account_email.gsub(/@/, "+#{Time.now.to_i}@")
+
+    gmail.send(
+      "signup@wifi.service.gov.uk",
+      test_email,
+      "",
+      "",
+    )
+
+    body = nil
+
+    Timeout.timeout(20, nil, "Waited too long for signup email") do
+      loop do
+        if (message = gmail.read("is:unread to:#{test_email}"))
+          body = message&.payload&.parts&.first&.body&.data
+          break
+        end
+
+        print "."
+        sleep 1
+      end
+    end
+
+    identity = body.scan(/Your username: ([a-z]+)/)&.first&.first
+    password = body.scan(/Your password: ([a-zA-Z]+)/)&.first&.first
+
+    expect(identity).to be
```

Comment:
Does this not need an argument?

Reply:
No - without an argument it just verifies it's set. We could test for a matching string based on expected regex.

Reply:
Actually we're already doing the regex in the scan above, so `be` should be sufficient. The regexes could be better though...